routes/auth-routes.js

```diff
@@ -1,140 +0,0 @@
-// The routes file will be defined in the routes/auth.routes.js, and we will set the necessary packages and code to signup in the application:
-
-const { Router } = require("express");
-const router = new Router();
-const passport = require("passport");
-const User = require("../models/user");
-const bcrypt = require("bcrypt");
-const bcryptSalt = 10;
-const { isLoggedIn, isLoggedOut } = require("../middleware/route-guard");
-const filteredDomains = require("../public/javascripts/filteredDomains");
-const nodemailer = require("nodemailer");
-const nodemailerSetup = require('../public/javascripts/nodemailer');
-
-// SIGNUP ROUTES, implemented with server-side validation on the email and password used. Users are also issued a verification code over email to verify their sign up.
-
-router.get("/signup", isLoggedOut, (req, res, next) => res.render("auth/signup", { layout: false }));
-
-router.post("/signup", isLoggedOut, (req, res, next) => {
```

Comment:
If you've read any of my other comments you know the drill now haha

Line breaks are super important, readability, etc, etc. This is a huge route and super difficult to follow the thread of what's going on. If I have to debug an error in this route, it's going to take me 15 mins to find the journey through this function, as opposed to 5 if it's formatted well.